gqlgen.yml

```diff
@@ -1,11 +1,12 @@
 # Where are all the schema files located? globs are supported eg  src/**/*.graphql
 schema:
-  - serve/graph/*.graphql
+  - serve/graph/schema/*.graphql
+  - serve/graph/schema/**/*.graphql
 
 # Where should the generated server code go?
 exec:
-  filename: serve/graph/generated.go
-  package: graph
+  filename: serve/graph/generated/generated.go
+  package: generated
```

Comment:
`setup.go` needs to import `resolver` and `generated`, and `resolver` imports `generated`. 
I separated setup and generated into separate packages because putting them at the same depth would cause import errors.

Reply:
If `resolver.go` and `generated.go` are on the same package as before `graph` then a cyclic import is not possible. `resolver.go` and `generated.go` does not need to import anything because they are on the same package, and `setup.go` can import graph, having methods from both without any error.

Reply:
You're right. 
If the three are managed in the same package, you won't get the cycle import error. 
Do you think it's good to have `resolver`, `genertated`, and `setup` managed in the same package?

Reply:
yes, as it was before if I'm remembering correctly, right?

Reply:
Yes, that's correct.
Then I'll change it back to managing them in one package.

I think it's better to keep them separate, but I'll think about it more.